CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ But you should always prefer just using
 const { DynamicThreadPool } = require('poolifier')
 ```
 
+#### New type definitions for input data and response
+
+For cluster and thread pools, you can now only send and receive serializable `JSON` data.
```

Comment:
I would add that this is due to a nodejs limitation and does not depends by this library.